templates/.pre-commit-config.local.yaml.jinja

```diff
@@ -0,0 +1,20 @@
+{% extends 'templates/.pre-commit-config.yaml.jinja' %}
+
+{% block note %}
+{{ super() }}
+# - https://github.com/WordPress/openverse/blob/main/templates/.pre-commit-config.local.yaml.jinja
+{%- endblock %}
+
+{% block repos_bottom %}
+{# This comment adds a blank line. #}
+  - repo: local
+    hooks:
+    {% set renders = ["precommit", "prettier", "github"] %}
+    {% for render in renders %}
```

Comment:
This is an amazing use of Jinja, I love it!

Reply:
I wish we could make even the list itself dynamic so we didn't have to update it, but alas that seems more challenging than it's worth 😅